docs/streams/developer-guide/dsl-api.html

```diff
@@ -3134,6 +3192,49 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
                 </div>
             </div>
         </div>
+	
+	<div class="section" id="controlling-update-rate">
+            <span id="streams-developer-guide-dsl-suppression"></span><h2><a class="toc-backref" href="#id32">Controlling KTable update rate</a><a class="headerlink" href="#controlling-update-rate" title="Permalink to this headline"></a></h2>
+	    <p>A KTable is logically a continuously updated table.
+	       These updates make their way to downstream operators whenever new data is available, ensuring that the whole computation is as fresh as possible.
+	       Logically speaking, most programs describe a series of transformations, and the update rate is not a factor in the program behavior.
+	       In these cases, the rate of update is more of a performance concern.
+	       Operators are able to optimize both the network traffic (to the Kafka brokers) and the disk traffic (to the local state stores) by adjusting
+	       the commit interval and batch size configurations.
+	    </p>
+	    <p>However, some applications need to take other actions, such as calling out to external systems, 
+	       and therefore need to exercise some control over the rate of invocations, for example of <pre>KStream#foreach</pre>.
+	    </p>
+	    <p>Rather than achieving this as a side-effect of the  <a class="reference internal" href="memory-mgmt.html#streams-developer-guide-memory-management-record-cache"><span class="std std-ref">KTable record cache</span></a>,
+	       you can directly impose a rate limit via the <pre>KTable#suppress</pre> operator.
+	    </p>
+	    <p>For example:
+	    </p>
+	    <div class="highlight-java"><div class="highlight">
+<pre>
+KGroupedTable&lt;String, String$gt; groupedTable = ...;
+groupedTable
+    .count()
+    .suppress(untilTimeLimit(ofMinutes(5), maxBytes(100_000_000L).emitEarlyWhenFull()))
+    .toStream()
+    .foreach((key, count) -$gt; updateCountsDatabase(key, count));
+</pre>
+	    </div></div>
+	    <p>This configuration ensures that <pre>updateCountsDatabase</pre> gets events for each <pre>key</pre> no more than once every 5 minutes.
+	       Note that the latest state for each key has to be buffered in memory for that 5-minute period.
+	       You have the option to control the maximum amount of memory to use for this buffer (in this case, 100MB).
```

Comment:
nit `100_000_000` != 100MB (only roughly the same)

Reply:
you're right, I was hoping to slide that by everyone ;) 
Seriously, I was being approximate in favor or readability, but if you think this will make everyone think we don't know about computers, I can change it.

I was thinking the base10 number is more readable, but this is technically MiB instead of MB. I don't know if I want to bring up MiB, though...

I guess I'll just make the number `104_857_600`?

Reply:
Or just say `about 1MB` ;)

Reply:
Is this not the same problem... `1_000_000` is not `1MB`?

Reply:
Waiiit a minute, I looked it up this time:
* 100,000,000 bytes is 100MB, and 1,000,000 is 1MB: MB is SI notation, so we use powers of 10.
* 104,857,600 bytes is 100MiB, and 1,048,576 is 1MiB: Mebibytes being the binary equivalent of Megabytes

Unless I got it backwards... This is what you were getting at initially?